csharp/ql/test/library-tests/overrides/Overrides22.expected

```diff
@@ -36,15 +36,15 @@
 | overrides.C3<>.Item[int] | overrides.I4.MyIndexer[int] | implements |
 | overrides.C3<>.Method() | overrides.I4.Method() | implements |
 | overrides.C3<>.Prop | overrides.I3.Prop | implements |
-| overrides.C3<Int32>.Item[int] | overrides.I4.MyIndexer[int] | implements |
-| overrides.C3<Int32>.Method() | overrides.I4.Method() | implements |
-| overrides.C3<Int32>.Prop | overrides.I3.Prop | implements |
+| overrides.C3<System.Int32>.Item[int] | overrides.I4.MyIndexer[int] | implements |
+| overrides.C3<System.Int32>.Method() | overrides.I4.Method() | implements |
+| overrides.C3<System.Int32>.Prop | overrides.I3.Prop | implements |
 | overrides.D.ToString() | overrides.C.ToString() | overrides |
 | overrides.D.f2() | overrides.A.f2() | overrides |
 | overrides.E2.M() | overrides.E.M() | overrides |
 | overrides.E.M() | overrides.I.M() | implements |
 | overrides.F.M() | overrides.I.M() | implements |
 | overrides.G2.M<S>(string, S) | overrides.G.M<S>(string, S) | overrides |
-| overrides.G.M<S>(string, S) | overrides.I2<String>.M<S>(string, S) | implements |
-| overrides.H<>.M<S>(TA, S) | overrides.I2<TA>.M<S>(TA, S) | implements |
-| overrides.Outer<>.A10.M<T>(Inner) | overrides.Outer<>.I6.M<T>(Inner) | implements |
+| overrides.G.M<S>(string, S) | overrides.I2<System.String>.M<S>(string, S) | implements |
+| overrides.H<>.M<S>(TA, S) | overrides.I2<overrides.H<>.<0>>.M<S>(TA, S) | implements |
```

Comment:
I'm surprised to see `TA` being changed to `overrides.H<>.<0>`, but `S` not being changed. Also, shouldn't all instances of `TA` be replaced by `overrides.H<>.<0>`?